platform/mds/mds-web/src/main/resources/webapp/js/directives.js

```diff
@@ -502,6 +502,9 @@
             link: function (scope, element, attr, ngModel) {
                 var isReadOnly = scope.$eval(attr.ngReadonly);
                 if(!isReadOnly) {
+                    angular.element(element).click( function() {
```

Comment:
The onclick function is assigned to input, not to "now" button, so after clicking on input the function search for that button and it does not switch classes but removes the "secondary" and add "primary" one. The "secondary" class is assigned to "Now" button by default in JQuery.